integration/core/test_api.py

```diff
@@ -276,42 +276,39 @@ def test_v2_upgrade(client):
 
 
 def test_upgrade_filters(client):
-    templates = client.list_template(catalogId='rancher')
+    templates = client.list_template(catalogId='qa-catalog')
     if len(templates) > 0:
         for i in range(len(templates)):
-            if templates[i].name == 'Kubernetes' \
-                    and templates[i].templateBase == 'infra':
+            if templates[i].id == unicode('qa-catalog:many-versions'):
                 versionUrlsMap = templates[i].versionLinks
-        assert len(versionUrlsMap) == 12
+        assert len(versionUrlsMap) == 15
```

Comment:
This makes the test brittle to changes in the catalog, just remove the assertion. its not particular important to the test, right?